include/arbiter/Requirement.h

```diff
@@ -32,9 +39,39 @@ typedef enum
 } ArbiterRequirementStrictness;
 
 /**
- * Represents a requirement for a specific version or set of versions.
+ * How suitable a specific version is for a requirement.
  */
-typedef struct ArbiterRequirement ArbiterRequirement;
+typedef enum
+{
+  /**
+   * The version is unsuitable for (does not satisfy) the requirement.
+   */
+  ArbiterRequirementSuitabilityUnsuitable,
+
+  /**
+   * The version is suitable for (satisfies) the requirement.
+   */
+  ArbiterRequirementSuitabilitySuitable,
+  
+  /**
+   * The version should be considered the best possible choice for satisfying
+   * the requirement.
+   *
+   * This result is unique in that it _overrides other requirements_, which is
```

Comment:
Not to get caught up on naming, but I like the word "override", or something that communicates that this has been hand picked.